beacon_node/client/src/builder.rs

```diff
@@ -251,12 +248,13 @@ where
             .ok_or_else(|| "node timer requires a chain spec".to_string())?
             .milliseconds_per_slot;
 
-        let timer_exit = context
-            .runtime_handle
-            .enter(|| timer::spawn(beacon_chain, milliseconds_per_slot))
-            .map_err(|e| format!("Unable to start node timer: {}", e))?;
-
-        self.exit_channels.push(timer_exit);
+        let _ = timer::spawn(
```

Comment:
Remember to be careful with the `let _ = `
Why are we dropping the result here?

Reply:
The spawn function returns `()`. Not sure what you mean.

Reply:
If the spawn function returns (), what is the point of `let _ = ` ?